src/azure-cli-core/azure/cli/core/tests/test_generic_update.py

```diff
@@ -7,10 +7,10 @@
 import unittest
 import sys
 
-from azure.cli.application import APPLICATION, Application, Configuration
-from azure.cli.commands import CliArgumentType, register_cli_argument
-from azure.cli.commands.arm import cli_generic_update_command
-from azure.cli._util import CLIError
+from azure.cli.core.application import APPLICATION, Application, Configuration
+from azure.cli.core.commands import CliArgumentType, register_cli_argument
+from azure.cli.core.commands.arm import cli_generic_update_command
```

Comment:
>.arm import cli_generic_update_command [](start = 28, length = 38)

I know this has nothing to do with your PR per se, but why is generic update part of arm.py? @johanste?